packages/api/src/external/carequality/process-outbound-patient-discovery-resps.ts

```diff
@@ -30,9 +38,10 @@ export async function processOutboundPatientDiscoveryResps({
   const patientIds = { id: patientId, cxId };
 
   try {
+    // BUG This prevents analytics from firing for valid cases of no links
     if (results.length === 0) {
       log(`No patient discovery results found.`);
-      await processPatientDiscoveryProgress({ patient: patientIds, status: "completed" });
+      await updatePatientDiscoveryStatus({ patient: patientIds, status: "completed" });
```

Comment:
Don't we need to also reset scheduled DQ if any? Not sure, TBH.

Reply:
Yea, I think you're right. I was going to remove this anyway as the code can handle no results and it skips analytics, but decided to leave it for now.

Reply:
Fixed.

Reply:
I was thinking even if there's no results, the user has requested DQ, so we shouldn't block that. There may be new documents on the links we have independent of whether we got new links from PD.

Reply:
Decision from call: always proceed to DQ (note: need to check schedule here as well(.

Reply:
Done.